apps/E2E/src/Chip/pages/ChipPageObject.ts

```diff
@@ -0,0 +1,44 @@
+import { BasePage, By } from '../../common/BasePage';
+import { AndroidAttribute } from '../../common/consts';
+import { CHIP_TESTPAGE, CHIP_TEST_COMPONENT, CHIP_TEXT, HOMEPAGE_CHIP_BUTTON } from '../consts';
+
+class ChipPageObject extends BasePage {
+  async getPrimaryComponentAttribute(attribute: string): Promise<string> {
```

Comment:
I think we should use `getElementAttribute()` from BasePage rather than create component-specific getters on each page.

That way, if we need to add more test elements in the future, we can normalize it all to `getElementAttribute()` rather than `getPrimaryComponentAttribute()`, `getSecondaryComponentAttribute()`, `getXComponentAttribute(), etc.

Reply:
I understand this might not be the pattern being used elsewhere, but in all new page objects, we should just use `getElementAttribute()`. Let's improve the code as we move forward

Reply:
Had retained these functions based on pattern followed in the BasicBadgePageObject, they aren't in active use though, removing from both places